gcp/README.md

```diff
@@ -0,0 +1,194 @@
+
+### Required GCP APIs
+
+The following APIs will be enabled:
+
+* Compute Engine API
+* Kubernetes Engine API
+* Cloud SQL Admin API
+* Secret Token API
+* Stackdriver Logging API
+* Stackdriver Monitoring API
+* IAM Service Account Credentials API
+
+All the tools for the demo are installed. When using Cloud Shell execute the following command in order to setup gcloud cli. When executing this command please setup your region and zone.
+
+```console
+gcloud init
+```
+
+### Tools
+
+When not using Cloud Shell, the following tools are required:
+
+* Access to an existing Google Cloud project.
+* Bash and common command line tools (Make, etc.)
+* [Terraform v0.14.0+](https://www.terraform.io/downloads.html)
+* [gcloud v333.0.0+](https://cloud.google.com/sdk/downloads)
+* [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) that matches the latest generally-available GKE cluster version.
+
+#### Install Terraform
+
+Terraform is used to automate the manipulation of cloud infrastructure. Its [Terraform installation instructions](https://www.terraform.io/intro/getting-started/install.html) are also available online.
+
+#### Install Cloud SDK
+
+The Google Cloud SDK is used to interact with your GCP resources. [Google cloud SDK Installation instructions](https://cloud.google.com/sdk/downloads) for multiple platforms are available online.
+
+#### Install kubectl CLI
+
+The kubectl CLI is used to interteract with both Kubernetes Engine and Kubernetes in general. [kubectl CLI Installation instructions](https://cloud.google.com/kubernetes-engine/docs/quickstart) for multiple platforms are available online.
+
+
+### Authenticate gcloud
+
+Prior to running this demo, ensure you have authenticated your gcloud client by running the following command:
+
+```console
+gcloud auth login
+```
+
+### Configure gcloud settings
+
+Run `gcloud config list` and make sure that `compute/zone`, `compute/region` and `core/project` are populated with values that work for you. You can choose a [region and zone near you](https://cloud.google.com/compute/docs/regions-zones/). You can set their values with the following commands:
+
+```console
+# Where the region is us-central1
+gcloud config set compute/region us-central1
+
+Updated property [compute/region].
+```
+
+```console
+# Where the zone inside the region is us-central1-c
+gcloud config set compute/zone us-central1-c
+
+Updated property [compute/zone].
+```
+
+```console
+# Where the project name is "my-project-name"
+gcloud config set project "my-project-name"
+
+Updated property [core/project].
+```
+
+## Test this project locally
+
+```
+cd env/
+terraform init
+````
+After the terraform initialization is successful, create your workspace and by using the following command
+
+```
+terraform workspace new <WORKSPACENAME>
+```
+
+or select the workspace if there are any existing workspaces
+
+```
+terraform workspace select <WORKSPACENAME>
+```
+
+Update variables.tf and setup your project ID
```

Comment:
Please provide more instructions here. Assume the user has not idea what variables.tf even is.